packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -105,6 +107,16 @@ export const createDefaultCallingHandlers: CreateDefaultCallingHandlers = memoiz
     ): Promise<void> => {
       const participant = _toCommunicationIdentifier(userId);
       await call?.removeParticipant(participant);
+    },
+    /* @conditional-compile-remove(spotlight) */
+    onStartSpotlight: async (userId: string): Promise<void> => {
```

Comment:
this exists already in createCommonHandlers -- is it needed here as well?

Reply:
Ah yes not needed here. Thanks.